crates/ruff_linter/src/rules/flake8_pytest_style/rules/assertion.rs

```diff
@@ -384,7 +384,11 @@ pub(crate) fn unittest_raises_assertion(
         },
         call.func.range(),
     );
-    if !checker.indexer().has_comments(call, checker.locator()) {
+    if !checker
```

Comment:
Nit: You could have kept the `has_comments` method on `Indexer` and simply forwarded the call to `CommentRanges` to reduce the diff size (It also increases encapsulation because callers don't need to know how the Indexer represents the data internally)

Reply:
I'm not sure if that makes any difference in terms of encapsulation because `CommentRanges` is already exposed via `indexer.comment_ranges` and calls to the other methods on `CommentRanges` are done directly.